### DISCUSSION TITLE
Is it possible that the cloning quality is much higher on the website than locally? Isn't the method supposed to be the same?

### DISCUSSION THREAD

**Discussion by Mixomo** [fishaudio/fish-speech#794] [Q&A]
Doing a voice cloning using the Fish Audio website, gives the expected results and the voice is identical to the audios provided. However, doing the same in local, the resulting voice has nothing to do. At most a slight resemblance with the timbre, but not much more. It does not respect the accent.
Is there something I am doing wrong?
How can I replicate the results of the web page, which are displayed as instant cloning? Do I need to do a local fine-tuning? I don't see much sense in it, since it would not be instant cloning...

Thanks in advance!

**Comment by thierry-rietsch** [comment]
I just experienced the same. Generated a cloned voice via web (which is pretty good) and locally. Unfortunately, the local generation is not really good.

**Reply by Mixomo** [reply]
Unfortunately, I have discovered through hugginface's demo comments that they use the "Fish Large" model And they left us with the fish medium model. That's why there's a difference in quality. They also said that they don't think they'll release the large model. 😒

**Reply by thierry-rietsch** [reply]
Oh.. good to know. Thanks for letting me know. Well then, off looking for another model that can be run locally 😀

**Reply by yuGAN6** [reply]
> Oh.. good to know. Thanks for letting me know. Well then, off looking for another model that can be run locally 😀

 I tried finetuning lora model with an hour speech data for each speaker, the quality is prettry good.  
looks like the released medium model is fast (150ms 1st package) but need fine-tuning for better quality.

for another model, cosyvoice2 local model is as good as its web demo.

**Reply by Mixomo** [reply]
Oh great! I still can't figure out/successfully train a LoRa on fish. 

How many hours were there in total approximately? 

Did you train the full cycle? (If I remember correctly, it's 10k steps?)

Did you use the command line or the web interface?

I am trying to train in Spanish. Perhaps for this language I need a lot more data than for English. 
I am training with a 3090 and the training does not give results, it takes too long. How long did your training take? 
Thanks!

**Reply by chesha1** [reply]
> Unfortunately, I have discovered through hugginface's demo comments that they use the "Fish Large" model And they left us with the fish medium model. That's why there's a difference in quality. They also said that they don't think they'll release the large model. 😒

Indeed. And I noticed that their paper doesn't include an ablation study on model size, nor do they mention models of other sizes in their Hugging Face model card or on GitHub. This could easily lead the community to assume that the released model is the only size they have. 😒

**Reply by SameplayerDE** [reply]
Always great when they release an open source model but then they use a model on their website as a demo that is not for local use and is much better

**Reply by La-Antique** [reply]
> > Oh.. good to know. Thanks for letting me know. Well then, off looking for another model that can be run locally 😀
> 
> I tried finetuning lora model with an hour speech data for each speaker, the quality is prettry good. looks like the released medium model is fast (150ms 1st package) but need fine-tuning for better quality.
> 
> for another model, cosyvoice2 local model is as good as its web demo.

hey would you mind sharing your results of your training?
